docs/azure-saas-docs/content/en/_index.md

```diff
@@ -1,36 +1,57 @@
+* It includes commonly needed SaaS components that implement features such as identity, onboarding, and tenant management.
+* It is 100% [Open-source code](https://github.com/Azure/azure-saas), allowing you to build by example or modify/extend to be purpose-built for your particular scenario.
+* The code is fully documented, making it clear how the code functions, and why key design decisions were made.
 
-Whether you're modernizing an existing application, building a new application, or migrating your application, the SaaS dev kit can help you.
+## Usage Options
 
-## Modules & Architecture
+How you use the dev kit is up to you, here are some ideas to get you started:
 
-![](futurestate.drawio.png)
+* Modernizing an existing application to support [full multitenancy](https://docs.microsoft.com/en-us/azure/architecture/guide/multitenant/considerations/tenancy-models#fully-multitenant-deployments) as part of a shift to a SaaS based business model.
+* Developing a greenfield SaaS offering for the first time.
+* Migrating a SaaS offering from another cloud to Azure.
 
-- [**B2C Authentication Service**](/services/b2c-auth-service/) - Provides a flexible identity solution.
-- [**Core App**](/services/core-app/)
-	- A web app where your customers to view plans and onboard to your solution.
-	- Provides you with tenant administration capabilities. (modify/remove/etc.)
-- [**SaaS.Application**](/services/saas-application/) - A sample application that you can extend or replace with your own code.
+This is not a one-size-fits-all solution. Use as little or as much as you like. It is designed to be both a modular deployable reference implementation and also a reference architecture. You are free to use and change the code contained within this project in any way you'd like (following the terms of the license):
 
-## Product SaaS Reference
+* Deploy the entire solution to Azure using the bicep templates we provide, make changes to fit your exact use case, and start building your SaaS application from there.
+* Deploy one or more of our modules and hook it into your existing SaaS application.
+* Reference our code & architecture and build something entirely custom using the best practices we outlined.
+* Or, simply gain inspiration from this codebase.
 
-[OnSubscriber](https://www.onsubscriber.com/) is a live production SaaS solution built entirely on the Azure SaaS Development Kit.
+## Modular Architecture
 
-It's a real production site that allows users to easily build their Email and SMS lists using social login with Facebook, Google, Apple, Email and SMS. In addition, users can add profile pictures, bios, external links and social accounts to their tenants.
+This kit uses a [microservices architecture](https://docs.microsoft.com/en-us/dotnet/architecture/microservices/architect-microservice-container-applications/microservices-architecture) so that each module is self-contained and can be used idependantly.
```

Comment:
```suggestion
This kit uses a [microservices architecture](https://docs.microsoft.com/en-us/dotnet/architecture/microservices/architect-microservice-container-applications/microservices-architecture) so that each module is self-contained and can be used independently.
```